scripts/daam_script.py

```diff
@@ -33,37 +33,38 @@ def title(self):
         return "Daam script"
 
     def show(self, is_img2img):
-        return True
+        return scripts.AlwaysVisible
 
     def ui(self, is_img2img):
-                
-        attention_texts = gr.Text(label='Attention texts for visualization. (comma separated)', value='')
+        with gr.Group():
+            with gr.Accordion("Attention Heatmap", open=False):
```

Comment:
This is the only really new part here. It wraps the contents in an accordion-style drop-down,